search-sentiments-map/public/script.js

```diff
@@ -75,3 +75,21 @@ function showMoreOrLess() {
     showMoreOrLessToggleItem.innerText = TOGGLE_SHOW_MORE;
   }
 }
+
+/** Gets the sentiment score of an inputted search topic. */
+function getSentiment() {
+  const searchTopic = document.getElementById('search-topic').value;
+  const sentimentScoreElement = document.getElementById('sentiment-score');
+  const searchTopicObject = "searchTopic="+encodeURIComponent(searchTopic);
+  fetch('/sentiment', {method: 'POST',  // Send a request to the URL.
+    headers: new Headers({
+      'Content-Type': 'application/x-www-form-urlencoded',
+    }),
+    body: searchTopicObject // Send data from index.html.
+    })
+    .then(response => response.json())
+    .then((score) => { 
+      console.log('ntarn debug: frontend' + score.sentimentScore);
```

Comment:
using this console log statement right now for getting the search results' sentiment score for a certain country

Reply:
same with other console.log statements

Reply:
Add a TODO for youself to remove this debug code since you wouldn't want it to be in the final version of this project.